src/libs/pr.ts

```diff
@@ -66,12 +81,14 @@ export async function handlePullRequestMessage(
 
   const [htmlBody, trimmed]: [string, boolean] = ansiToHtml(output, MAX_CHARACTER_COMMENT, alwaysIncludeSummary);
 
+  const viewLiveLink = extractViewLiveLink(output);
+
   const body = dedent`
     ${heading}
 
     <details>
     ${summary}
-
+    ${viewLiveLink ? `[View Live](${viewLiveLink})\n` : ''}
```

Comment:
Very minor bikeshedding here, but maybe 
```suggestion
    ${viewLiveLink ? `[View in Pulumi Cloud](${viewLiveLink})\n` : ''}
```
would be slightly clearer? When the comment is posted the update has already finished, so there's no viewing it "live" anymore.

Reply:
Yes I agree, that's better. I'll update things in a bit!